addons/html_editor/static/src/components/history_dialog/history_dialog.xml

```diff
@@ -1,121 +1,157 @@
-                            </div>
+            <div class="d-flex flex-column h-100 overflow-hidden">
+                <ul class="nav nav-tabs d-md-none flex-shrink-0" role="tablist">
+                    <li class="nav-item flex-grow-1">
+                        <a href="#" id="tab-revisions" class="nav-link position-relative text-center fw-bold"
+                            t-att-class="{'active': state.mobileActiveTab === 'revisions'}"
+                            t-on-click.prevent="() => this.state.mobileActiveTab = 'revisions'"
+                            role="tab"
+                            aria-controls="revisions-panel"
+                            t-att-aria-selected="state.mobileActiveTab === 'revisions' ? 'true' : 'false'">
+                            <i class="fa fa-list me-1"/>Revisions
                         </a>
-                    </t>
-                </div>
-                <div class="history-container" t-attf-style="max-height: {{state.cssMaxHeight}}px;">
-                    <div t-attf-class="history-content-view #{state.currentView === 'content' ? '' : 'd-none'}">
-                        <div class="history-view-top-bar">
-                            <div>
-                                <div class="text-info smaller">
-                                    <i class="fa fa-info-circle me-1" title="Version date"/>
-                                    Showing the document as it was on <t t-esc="this.getRevisionDate(this.currentRevision)" />, submited by <t t-esc="this.currentRevision.create_user_name" />
-                                </div>
+                    </li>
+                    <li class="nav-item flex-grow-1">
+                        <a href="#" id="tab-preview" class="nav-link position-relative text-center fw-bold"
+                            t-att-class="{'active': state.mobileActiveTab === 'content'}"
+                            t-on-click.prevent="() => this.state.mobileActiveTab = 'content'"
+                            role="tab"
+                            aria-controls="content-panel"
+                            t-att-aria-selected="state.mobileActiveTab === 'content' ? 'true' : 'false'">
+                            <i class="fa fa-eye me-1"/>Preview
+                        </a>
+                    </li>
+                </ul>
+
+                <div t-attf-class="html-history-dialog position-relative d-flex h-100 #{state.isLoading ? 'html-history-loading' : 'html-history-loaded'} #{this.getMobileDialogClass()}">
+                    <div id="revisions-panel" class="revision-list h-100 p-3 border-end overflow-auto" role="region" aria-labelledby="tab-revisions">
+                        <t t-if="!state.revisionsData.length">
+                            <div class="text-center w-100 pb-2 pt-0 px-0 fw-bolder">No history</div>
+                        </t>
+
+                        <ul class="list-group" role="listbox" t-ref="listbox" t-else="">
```

Comment:
Start with the condition so it's easier to understand it follows the t-if element above.
```suggestion
                        <ul t-else="" class="list-group" role="listbox" t-ref="listbox">
```